### PR TITLE
Update File

### PR DESCRIPTION
**Pull request by rifanrifai11**
Update File

### CODE REVIEW

File: hello.txt [hunk 1]
```diff
@@ -1,7 +1,2 @@
 hello world saja
```
Comment (other-team) [inline hello.txt:1]: Segera di review

**Review by other-team — COMMENTED** [review]
merge OK

**Review by ekojunirianto — COMMENTED** [review]
ok

File: hello.txt (hunk 1 above)
Comment (ekojunirianto) [inline hello.txt:1]: Ok

**Pull request merged**